doc/api/deprecations.md

```diff
@@ -2335,18 +2335,41 @@ Type: Runtime
 Setting the TLS ServerName to an IP address is not permitted by
 [RFC 6066][]. This will be ignored in a future version.
 
-<a id="DEP0XXX"></a>
-### DEP0XXX: using REPLServer.rli
+<a id="DEP0124"></a>
+### DEP0124: using REPLServer.rli
 <!-- YAML
 changes:
   - version: REPLACEME
     pr-url: https://github.com/nodejs/node/pull/26260
     description: Runtime deprecation.
+
+This property is a reference to the instance itself.
+
+<a id="DEP0125"></a>
+### DEP0125: accessing global.process in ECMAScript Modules
```

Comment:
I believe we do not have to go through normal deprecation process if this is limited to ESM, which is experimental anyways - it needed a doc-deprecation previously because it affected CJS.

Reply:
Great, will remove the code.